library/lua/gui/widgets/text_area.lua

```diff
@@ -101,15 +100,15 @@ end
 
 function TextArea:scrollToCursor(cursor_offset)
     if self.scrollbar.visible then
-        local _, cursor_liny_y = self.text_area.wrapped_text:indexToCoords(
+        local _, cursor_line_y = self.text_area.wrapped_text:indexToCoords(
             cursor_offset
         )
-        self:updateScrollbar(cursor_liny_y)
+        self:updateScrollbar(cursor_line_y)
     end
 end
 
 function TextArea:getPreferredFocusState()
-    return self.parent_view.focus
+    return self.parent_view and self.parent_view.focus or true
```

Comment:
why not just `true`?

Reply:
I read the docs, but I was always confused by this "preferred focus" feature. 
Changed to `true`